bootloader/bootloader.go

```diff
@@ -41,6 +41,10 @@ type Options struct {
 	// used at prepare-image time, that means not on a runtime
 	// system.
 	PrepareImageTime bool
+
+	// Recovery indicates use a recovery bootloader. Note that
+	// UC16/UC18 do not have a recovery system.
```

Comment:
// Recovery indicates to use the recovery bootloader. Note that 
// UC16/18 do not have a recovery partition.